.lintstagedrc.mjs

```diff
@@ -1,5 +1,5 @@
 // @ts-check
 export default {
-  '!(*.{js,jsx,ts,tsx,npmrc})': 'prettier --write',
+  '!(*.{js,jsx,ts,tsx,npmrc,gitignore})': 'prettier --write',
```

Comment:
Ran into a prettier parser error when I added the `.sanity` build folder to kitchensink's gitignore. I think this is correct?